oasis/recipe

```diff
@@ -7,16 +7,6 @@ Library recipe
   Path: lib/bap_recipe
   FindlibName: bap-recipe
   CompiledObject: best
-  BuildDepends: base, stdio, parsexp, fileutils, camlzip, uuidm
+  BuildDepends: stdlib-shims, base, stdio, parsexp, fileutils, camlzip, uuidm
```

Comment:
do we really need them?

Reply:
yep, we need, I've used the `Stdlib` module accidentally, mostly thinking that it is overridden by Base, but apparently I was wrong. Since we already indirectly depend on those shims, let's keep them. However, after BAP 2.0 will be finally released, we will drop them along with the older versions of the compiler and core_kernel.